docs/release.rst

```diff
@@ -61,6 +61,9 @@ Maintenance
 * Remove ``sphinx-rtd-theme`` dependency from ``pyproject.toml``.
   By :user:`Sanket Verma <MSanKeys963>` :issue:`1563`.
 
+* Remedy a situation where ``zarr-python`` was importing ``DummyStorageTransformer`` from the test suite. 
+  The dependency relationship is now reversed: the test suite imports this class from ``zarr-python``.
+  By :user:`Davis Bennett <d-v-b>` :issue:`1601`.
```

Comment:
Is this in the right place? We have not been doing release notes for the v3 branch and this seems like its mixed in with 2.17.

Reply:
oops, you are right! where should we put release notes for v3 stuff? :thinking:

Reply:
I've been thinking we'll do a review of the v3 branch and aggregate the meaningful things into the release notes based on the commit history and code review. But we can start adding notes now (just not where you put them 😉 ).

Reply:
thoughts on something like this? 

```
[normal release notes above]

Unreleased (v3)
---------------


* Remedy a situation where ``zarr-python`` was importing ``DummyStorageTransformer`` from the test suite. 
  The dependency relationship is now reversed: the test suite imports this class from ``zarr-python``.
  By :user:`Davis Bennett <d-v-b>` :issue:`1601`.

.. _unreleased:

[v2 release notes appended below]

Unreleased (v2)
---------------

Docs
~~~~

* Minor correction and changes in documentation.
  By :user:`Sanket Verma <MSanKeys963>` :issue:`1509`.

...
```

Reply:
lgtm

Reply:
added in https://github.com/zarr-developers/zarr-python/pull/1601/commits/8e790a73a622464f12cebd3fa097bb4599c3379d